var assert = require('assert');

assert.doesNotThrow(function(){
	var a = 1+2;
	console.log(a);
},"Their is no Error");
